docs/user_guide/shifts.md

```diff
@@ -0,0 +1,203 @@
+Consequently, this uncertainty can be integrated into a statistical inference model like this:
+
+```{literalinclude} ../../analysis_templates/cms_minimal/__cf_module_name__/inference/example.py
+:dedent:
+:language: python
+:start-at: '# lumi'
+:end-before: '# tune uncertainty'
+```
+
+(weight_based_uncertainties)=
+
+## Weight-based uncertainties
+
+This class of shifts consists of uncertainties that enter the analysis with the application of weights to events or individual objects.
+This way, they can modify both the yield and the general shape of the distribution.
+Popular examples for this type of uncertainty are scale factors to generally improve the compatibility between observed data and simulation before the measurement.
+Since these factors are usually measured in dedicated control regions, they are also subject to both statistical and systematic uncertainties that need to be taken into account in the final statistical inference.
+The following example is based on scale factors for muons.
+
+columnflow provides the {py:class}`~columnflow.production.cms.muon.muon_weights` Producer to calculate the weights for scale factors.
+This module creates three columns: one for weights derived from the nominal scale factors and columns where the scale factors are varied within the 68% confidence intervals of the measurement (up and down variations).
+In order to derive templates that correspond to these variations, we need to tell the workflow to use the varied versions of this weight when creating the templates.
+
+As explained in the [config section](building_blocks/config_objects.md#shift), the shift needs to be registered in the config.
+As demonstrated in the cms analysis example, this can be done like so:
+
+```{literalinclude} ../../analysis_templates/cms_minimal/__cf_module_name__/config/analysis___cf_short_name_lc__.py
+:dedent:
+:language: python
+:start-at: '# event weights due to muon scale factors'
+:end-before: '# add column aliases for shift mu'
+```
+
+Next, we define a column alias as auxiliary information for each of these shifts.
+This can later be used to switch out the name of a given column (e.g. `muon_weight`) with a varied version of the quantity (e.g. `muon_weight_up`).
+For convenience, we provide the {py:func}`~columnflow.config_util.add_shift_aliases` function to easily add the alias for both variations of the shift at once:
+
+```{literalinclude} ../../analysis_templates/cms_minimal/__cf_module_name__/config/analysis___cf_short_name_lc__.py
+:dedent:
+:language: python
+:start-at: '# add column aliases for shift mu'
+:end-before: '# external files'
+```
+
+Note that this function makes some assumptions on the naming of the varied quantities and the shifts.
+
+Finally, we can now configure the modules within the workflow to use this shift.
+As mentioned above, this class of systematic uncertainties does not change any selection efficiencies.
```

Comment:
The selection efficiency can be modified via event weights, but we can still reuse SelectEvents because the acceptance of any individual event is not changed.

```suggestion
As mentioned above, this class of systematic uncertainties does not affect the selection acceptance.
```